src/helpers/whitelist.ts

```diff
@@ -19,6 +19,22 @@ type CachedVerifyAddressResponse = VerifyAddressResponse & {
   cachedAt?: number;
 };
 
+const getParsedAddressList = (address: string | string[]) => {
+  if (typeof address === 'string') {
+    return isHaqqAddress(address)
```

Comment:
Lets create two helpers. toHaqq and toBech32. And use it here like:

` if (typeof address === 'string') { return toBech32(address) }`
and
```
if (Array.isArray(address)) {
    return address.map(toBech32);
  }
```

Otherwise we will duplicate this logic in other places